build.sh

```diff
@@ -4,50 +4,6 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
 
-# Set OFFLINE environment variable to build offline
-
-set -e
```

Comment:
`set -e` was my concern

---
In reply to: [115821558](https://github.com/dotnet/core-setup/pull/2342#discussion_r115821558) [](ancestors = 115821558)